docs/reference/indices/simulate-template.asciidoc

```diff
@@ -0,0 +1,308 @@
+[[indices-simulate-template]]
+=== Simulate index template API
+++++
+<titleabbrev>Simulate template</titleabbrev>
+++++
+
+experimental[]
+
+Returns the index settings that would be applied by a particular 
+<<indices-templates, index template>>.
+
+////
+[source,console]
+--------------------------------------------------
+PUT _component_template/component_template1
+{
+  "template": {
+    "mappings": {
+      "properties": {
+        "@timestamp": {
+          "type": "date"
+        }
+      }
+    }
+  }
+}
+
+PUT _component_template/other_component_template
+{
+  "template": {
+    "mappings": {
+      "properties": {
+        "ip_address": {
+          "type": "ip"
+        }
+      }
+    }
+  }
+}
+
+PUT _index_template/template_1
+{
+  "index_patterns": ["te*", "bar*"],
+  "template": {
+    "settings": {
+      "number_of_shards": 1
+    },
+    "mappings": {
+      "_source": {
+        "enabled": false
+      },
+      "properties": {
+        "host_name": {
+          "type": "keyword"
+        },
+        "created_at": {
+          "type": "date",
+          "format": "EEE MMM dd HH:mm:ss Z yyyy"
+        }
+      }
+    },
+    "aliases": {
+      "mydata": { }
+    }
+  },
+  "priority": 10,
+  "composed_of": ["component_template1", "other_component_template"],
+  "version": 3,
+  "_meta": {
+    "description": "my custom"
+  }
+}
+--------------------------------------------------
+// TESTSETUP
+
+[source,console]
+--------------------------------------------------
+DELETE _index_template/*
+DELETE _component_template/*
+--------------------------------------------------
+// TEARDOWN
+////
+
+[source,console]
+--------------------------------------------------
+POST /_index_template/_simulate/template_1
+--------------------------------------------------
+
+[[simulate-template-api-request]]
+==== {api-request-title}
+
+`POST /_index_template/_simulate/<index-template>`
+
+
+[[simulate-template-api-path-params]]
+==== {api-path-parms-title}
+
+`<index-template>`::
+(Optional, string)
+The name of the index template to simulate.
+To simulate an arbitrary template configuration, 
+omit this parameter and specify the template configuration in the request body.
```

Comment:
As before, I'd drop the first article from these defs.

I also wonder if `arbitrary template configuration` really gets at the intended use here:
`If you want to simulate an index template before creating it,...`
```suggestion
Name of the index template to simulate.
+
To simulate an arbitrary template configuration, 
omit this parameter and specify the template configuration in the request body.
```